src/styles/utils/_variables.scss

```
@@ -0,0 +1,15 @@
+$day-background-color: #eee;
+$day-background-color-hover: #ffbfcb;
+$day-width: 100px;
+$day-height: 100px;
+$gap: 1px;
+$column-number: 7;
+$total-gap-distance: $gap * ($column-number - 1);
+$calendar-width: ($day-width * $column-number) + $total-gap-distance;
+$start-day: 'mon' 0,
+  'tue' ($day-width) + ($gap),
+  'wed' ($day-width * 2) + ($gap * 2),
+  'thu' ($day-width * 3) + ($gap * 3),
+  'fri' ($day-width * 4) + ($gap * 4),
+  'sat' ($day-width * 5) + ($gap * 5),
+  'sun' ($day-width * 6) + ($gap * 6);
```

move calculations into margin-left to remove it's repetition 
```suggestion
  'tue' 1,
  'wed' 2,
  'thu' 3,
  'fri' 4,
  'sat' 5,
  'sun' 6;
```